data/json/recipes/electronics/parts.json

```diff
@@ -21,6 +21,24 @@
       [ [ "acid", 4 ], [ "fertilizer_liquid", 4 ], [ "fertilizer_commercial", 4 ] ]
     ]
   },
+  {
+    "result": "flux_comp_gen",
+    "type": "recipe",
+    "category": "CC_ELECTRONIC",
+    "subcategory": "CSC_ELECTRONIC_PARTS",
+    "skill_used": "electronics",
+    "difficulty": 6,
+    "skills_required": [ "fabrication", 4 ],
+    "time": "3h 20m",
+    "reversible": true,
+    "book_learn": [ [ "manual_electronics", 6 ] ],
+    "using": [ [ "soldering_standard", 10 ], [ "surface_heat", 50 ], [ "welding_standard", 5 ] ],
+    "qualities": [ { "id": "SCREW", "level": 1 } ],
+    "tools": [ [ [ "mold_plastic", -1 ] ] ],
+    "//": "Fill pipe with RDX, wind copper tubes into coil around it, leaving empty space. Use plastic (representing fiberglass) to insulate wires and supporting the coil of wires",
+    "//1": "plastic matches fiberglass required roughly by volume (4804mL), not by weight",
```

Comment:
Fiberglass and common polymers are completely different. All of the materials the paper describes for this use would be hard to work with for a survivor and we don't have most of them in the game. I don't think this should be craftable unless we add a load of other crafting stuff for working with those materials.

Reply:
Okay, removed the recipe.